autoscheduler/frontend/src/components/SchedulingPage/SchedulingPage.tsx

```diff
@@ -48,6 +57,23 @@ const SchedulingPage: React.FC<SchedulingPageProps> = ({
       </div>
       <div className={styles.scheduleContainer}>
         <Schedule />
+        <div
+          className={styles.fullscreenButtonContainer}
+          style={{ backgroundColor: theme.palette.primary.main }}
+        >
```

Comment:
imo this should use the `GenericCard` component and add an option to make it maroon (a `color: 'primary' | secondary'` prop?), it should also have rounded edges like all the other cards on the page

Reply:
I wouldn't be opposed to making a `GenericHeader` if you'd like, but I don't think `GenericCard` would be appropriate since it won't actually have any children (and thus won't be a card). I also have to edit the `GenericCard` a bit to get it to work how we want it to be, b/c we have to change the border radius and height of its internal `header` div, which isn't possible currently (basically just also need to add the ability to change the styles of the header, currently it's just for the card content). Not that I mind doing it ofc, I just think it's going beyond the functionality that a `GenericCard` needs in most other cases and as such it should be something different probably.

I do agree it should have rounded edges, even left a comment to remind myself but I forgot lol. I'll add them now and we can follow up about making this generic later if you have some more ideas on it